source/gui/__init__.py

```diff
@@ -200,6 +200,18 @@ def onTemporaryDictionaryCommand(self,evt):
 		# Translators: Title for temporary speech dictionary dialog (the voice dictionary that is active as long as NvDA is running).
 		self._popupSettingsDialog(DictionaryDialog,_("Temporary dictionary"),speechDictHandler.dictionaries["temp"])
 
+	def onExecuteUpdateCommand(self, evt):
+		if updateCheck and updateCheck.isPendingUpdate():
+			updateCheck.executeUpdate()
+
+	def onUpdatePendingUpdateMenuItemCommand(self, evt):
+		try:
+			self.sysTrayIcon.menu.RemoveItem(self.sysTrayIcon.runPendingUpdateMenuItem)
+		except:
+			pass
+		if updateCheck and updateCheck.isPendingUpdate():
+			self.sysTrayIcon.menu.InsertItem(self.sysTrayIcon.menu.GetMenuItemCount()-2,self.sysTrayIcon.runPendingUpdateMenuItem)

```

Comment:
What's with the `-2`. Is this the position. Might be clearer to put this in a `pendingUpdateMenuItemPosition` variable. Is it important for it to be second from the bottom? I am concerned that because this positioning is separated from the rest of that menu filing that the order might change, and this code wont be considered. The index of the position for this item could be saved at the point where the rest of the menu is populated.